kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -90,6 +94,26 @@ local networkSpec = networkPolicy.mixin.spec;
                     // model-server doesn't have something we can use out of the box.
                     resources: defaults.resources,
                   },
+		  if httpProxyImage != 0 then
+		  {
+		    name: name+"-http-proxy",
+                    image: httpProxyImage,
+                    imagePullPolicy: defaults.imagePullPolicy,
+                    command: [
+                      "/bin/sh",
```

Comment:
Why do we we have to wrap the command in a shell? Why can't we just do

command: [
  "python",
  "/usr/src/app/server.py",
   ...
]